Source/RapyutaSimulationPlugins/Private/Robots/RobotVehicle.cpp

```diff
@@ -14,15 +14,15 @@
 
 ARobotVehicle::ARobotVehicle()
 {
-    SetupDefault();
+    SetupRootSkeletal();
 }
 
 ARobotVehicle::ARobotVehicle(const FObjectInitializer& ObjectInitializer) : Super(ObjectInitializer)
 {
-    SetupDefault();
+    SetupRootSkeletal();
 }
 
-void ARobotVehicle::SetupDefault()
+void ARobotVehicle::SetupRootSkeletal()

```

Comment:
I'd suggest `SetupDefault()` name be kept since it literally signify the function's meaning as to create default child objects/components that are exposed to BP users so they could config them statically.

Reply:
I use SetupDefault() in the parent class and it creates the default object there.
SetupRootSkeletal() only creates the skeletalMesh and sets it as the root component so I renamed it

Reply:
Ah right, sorry, since the function is supposed to only be called inside ctor, thus could not be made virtual, so maybe it should be `SetupDefaultRootSkeletal()` just for clarity.